app/src/main/java/is/xyz/mpv/MPVLib.kt

```diff
@@ -0,0 +1,176 @@
+package `is`.xyz.mpv
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.view.Surface
+
+// Wrapper for native library
+
+@Suppress("unused")
+object MPVLib {
+    init {
+        val libs = arrayOf<String?>("mpv", "player")
+        for (lib in libs) {
+            System.loadLibrary(lib)
+        }
+    }
+
+    external fun create(appctx: Context)
+    external fun init()
+    external fun destroy()
+    external fun attachSurface(surface: Surface)
+    external fun detachSurface()
+
+    external fun command(cmd: Array<String>)
```

Comment:
I think this is correct?
```suggestion
    external fun command(cmd: Array<out String>)
```

Reply:
Both would work, but if you want to accept subtypes of `String` like `Nothing`, which... I don't really find a use for in this case, then yes, the `Array<out String>` would also be correct